pendant/readme.txt

```diff
@@ -42,3 +42,39 @@ This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 GNU General Public License for more details.
+
+This theme bundles the following third-party resources:
+
+MET Museum
+License : Public Domain
+Source  : https://www.metmuseum.org/art/collection/search/243039
+          https://www.metmuseum.org/art/collection/search/248813
+          https://www.metmuseum.org/art/collection/search/452273
+	  https://www.metmuseum.org/art/collection/search/256970
+	  https://www.metmuseum.org/art/collection/search/236772
+	  https://www.metmuseum.org/art/collection/search/242952
```

Comment:
last 3 are not used in the theme. 
Also, for first 3 png versions are used in the theme. we could clean jpeg images from assets.